# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/nsIDragService.idl
# Commit: beeda5ef3712
# Full Hash: beeda5ef3712b73a19c7a4ce08a52ab4dd08b802
# Author: David P <daparks@mozilla.com>
# Date: 2024-07-04 16:09:52
# Regressor Bug: 1893119
# File Overlap Count: 1
# Description:
#   Bug 1893119: Part 20 - Move EndDragSession from nsIDragService to nsIDragSession r=gstoll,rkraesig,win-reviewers,geckoview-reviewers,m_kato
#   
#   This is a straightforward move.  It does take the liberty of breaking out an
#   EndDragSessionImpl method, which will be needed later.
#   
# ==============================================================================

diff -r 6c3ffc8c534c -r beeda5ef3712 widget/nsIDragService.idl
--- a/widget/nsIDragService.idl	Thu Jul 04 07:48:10 2024 +0000
+++ b/widget/nsIDragService.idl	Thu Jul 04 07:48:11 2024 +0000
@@ -174,17 +174,6 @@
                                 in unsigned long aAllowedEffect);
 
   /**
-    * Tells the Drag Service to end a drag session. This is called when
-    * an external drag occurs
-    *
-    * If aDoneDrag is true, the drag has finished, otherwise the drag has
-    * just left the window.
-    */
-  [can_run_script]
-  void endDragSession(in boolean aDoneDrag,
-                      [optional] in unsigned long aKeyModifiers);
-
-  /**
    * Increase/decrease dragging suppress level by one.
    * If level is greater than one, dragging is disabled.
    */